packages/react-button/src/components/Button/stories/ButtonDefault.stories.tsx

```diff
@@ -2,5 +2,5 @@ import * as React from 'react';
 import { Button, ButtonProps } from '../../../Button';
 
 export const Default = (props: ButtonProps) => {
-  return <Button {...props}>Button</Button>;
+  return <Button {...props}>Default</Button>;
```

Comment:
We could leave it as "Example", part of this PR is removing the word "button" from the `Button` text because it sounds weird in screen readers due to repetition.